setup.cfg

```diff
@@ -20,11 +20,11 @@ classifiers =
 [options]
 # We set packages to find: to automatically find all sub-packages
 packages = find:
-requires = numpy
+requires = numpy>=1.19.3  # keep in sync with NPY_TARGET_VERSION (setup.py)
 zip_safe = False
 tests_require = pytest-doctestplus
 setup_requires = setuptools_scm
-install_requires = numpy>=1.19
+install_requires = numpy>=2.0.0rc1
```

Comment:
this line seems entirely redundant with `pyproject.toml`, however, it's been a while since I actually used `setup.cfg` so I'm only 99% confident. If that's desired, I'm game to pull the band aid and merge those two files together (which is essentially automated with the right tooling)

Reply:
Let's do that in a separate PR...

Reply:
We don't want to pin numpy>2.0rc1 in ``install_requires`` - this is for runtime dependencies!

Reply:
(this is different from ``setup_requires`` which is the old way of specifying build-time dependencies)

Reply:
Oh my you're right. I meant to touch the previous line and didn't realize my mistake in review... will fix !